workers/unity/Assets/Fps/Scripts/Editor/GameLogic/WorldTiles/MapQualitySettingsInspector.cs

```diff
@@ -5,7 +5,7 @@
 namespace Fps
 {
     [CustomEditor(typeof(MapQualitySettings))]
-    public class MapQualitySettingsInspector : Editor
+    public class MapQualitySettingsInspector : UnityEditor.Editor
```

Comment:
is that change needed?

Reply:
yes, otherwise it complains about missing references